src/main/java/at/hannibal2/skyhanni/config/features/garden/SensReducerConfig.java

```diff
@@ -0,0 +1,31 @@
+package at.hannibal2.skyhanni.config.features.garden;
+
+import at.hannibal2.skyhanni.config.core.config.Position;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class SensReducerConfig {
+    @Expose
+    @ConfigOption(
+        name = "Enable",
+        desc = "Lowers mouse sensitivity while in the garden, and a farming tool is held.")
+    @ConfigEditorBoolean
+    public boolean enabled = false;
+
+    @Expose
+    @ConfigOption(name = "Reducing factor", desc = "Changes by how much the sensitivity is lowered by.")
+    @ConfigEditorSlider(minValue = 0, maxValue = 100, minStep = 1)
+    public int divisorSens = 25;
```

Comment:
You wrap this int value with a Property, like at https://github.com/hannibal002/SkyHanni/blob/57acdfea63a58203cebda34ef7e16ed6fcc1bf1d/src/main/java/at/hannibal2/skyhanni/config/features/dev/RepoPatternConfig.java#L12